src/lib/php/Test/TestPgDb.php

```diff
@@ -178,7 +178,7 @@ public function createInheritedTables($tableList=array())
   {
     $table = 'license_candidate';
     if ((empty($tableList) || in_array($table, $tableList)) && !$this->dbManager->existsTable($table)) {
-      $this->dbManager->queryOnce("CREATE TABLE $table (group_fk integer) INHERITS (license_ref)");
+      $this->dbManager->queryOnce("CREATE TABLE $table (group_fk integer,rf_creationdate timestamptz,rf_lastmodified timestamptz,rf_user_fk_created text,rf_user_fk_modified text) INHERITS (license_ref)");
```

Comment:
Shouldn't `rf_user_fk_created` and `rf_user_fk_modified` be integers and not text to match the `core-schema.dat`?

Reply:
okay

Reply:
so I have to change its data type?

Reply:
Yes, the test data should match as closely as possible to production in order to provide optimum test coverage.

Reply:
i have changed it to varchar

Reply:
Okay but the core-schema says it should be int4????